include/envoy/server/admin.h

```diff
@@ -18,7 +19,9 @@ namespace Server {
  * done in the RouteConfigProviderManagerImpl constructor in source/common/router/rds_impl.cc.
  */
 #define MAKE_ADMIN_HANDLER(X)                                                                      \
-  [this](const std::string& url, Buffer::Instance& data) -> Http::Code { return X(url, data); }
+  [this](const std::string& url, Http::HeaderMap& header, Buffer::Instance& data) -> Http::Code {  \
```

Comment:
nit: s/headers/response_headers (same below in other cases)

Reply:
done